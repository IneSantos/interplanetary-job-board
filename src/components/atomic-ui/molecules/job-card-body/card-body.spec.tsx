import React from "react";
import { render } from '@testing-library/react';
import CardBody from "./card-body";

describe("CardBody atom tests", () => {
  test(`CardBody component render`, () => {
    const { container } = render(
      <CardBody skills={[]} description="This is a description"/>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
