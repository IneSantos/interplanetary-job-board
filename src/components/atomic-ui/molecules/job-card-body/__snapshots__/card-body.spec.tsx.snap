// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBody atom tests CardBody component render 1`] = `
<div
  class="card-body"
>
  <div
    class="description"
  >
    This is a description
  </div>
  <div
    class="skill-list"
  />
  <a
    class="job-apply"
    href=""
  >
    Apply
  </a>
</div>
`;
